tests/unit/api/main.py

```diff
@@ -588,7 +602,7 @@ def test_node_update_metadata(self):
         metadata = api._must_find_n(api._must_find(model.Node,
                                                    'runway_node_0'),
                                     model.Metadata, 'EK')
-        assert metadata.value == 'new_pk'
+        assert metadata.value == json.dumps('new_pk')
```

Comment:
I'd do this the other way around: `assert json.loads(metadata.value) == 'new_pk'`. This is a slightly weaker constraint which isn't sensitive to semantically irrelevant details like whitespace. It should work fine anyway as long as we're using the same marshalling library, but I think it's a bit cleaner the other way.